### ISSUE TITLE
Russian verison uses Cataclysm text

### ISSUE THREAD

**Issue by Defalion** [azerothcore/azerothcore-wotlk#15135]
### Current Behaviour

Hi, ive noticed that some of the text in rRussian language mentions Deathwing or other Cataclysm related text.
For an example Deputy William at Northshire mentions Deathwing at his first quest https://www.wowhead.com/wotlk/quest=783/a-threat-within
Another example will be mentioning Balckrock Orcs on the https://www.wowhead.com/wotlk/item=745/marshal-mcbrides-documents. 
No other examples yet, but i think that some text is related to expansions way past WotLK. 
+ Sometimes you can see Deathwing on the log in screen
++ Some of the dialogues and quests are in English, no matter the Russian language settings

### Expected Blizzlike Behaviour

I belive that Deathwing should not be mentioned in WotLK

### Source

My single player game.

### Steps to reproduce the problem

I used Azeroth Core with Single player project. 
Just create a new human character in russian language and it will be seen

### AC rev. hash/commit

AzerothCore rev. eb36cbb42c08 2023-02-18 08:51:19 +0700 (npcbots_3.3.5 branch) (Win64, RelWithDebInfo, Static) (worldserver-daemon)

### Operating system

Windows 10

### Custom changes or Modules

dungeonrespawn
Gainhonorguard
individualprogression
mod_achievements
mod_ahbot
mod_LuaEngine
SoloLfg

**Comment by heyitsbench** [comment]
> Sometimes you can see Deathwing on the log in screen

Then your client is to blame.

> Some of the dialogues and quests are in English, no matter the Russian language settings

Any points of this you can find, feel free to open an issue for, assuming one doesn't already exist for it.

**Comment by Defalion** [comment]
i had people on az core discord say its database issue

**Comment by Gultask** [comment]
The login screen being from Cataclysm *is* a client issue. The texts aren't.

**Comment by heyitsbench** [comment]
> i had people on az core discord say its database issue or core

The point I quoted (Deathwing on the login screen) *is* a client issue. The rest of it is almost definitely localization issues inside the DB. The second part of my statement was saying that those issues should be reported individually, as big issues are harder to keep track of.

**Comment by Defalion** [comment]
well this ticket reports a **translation bug** - ive added mention of log in screen just as **additional info** that might be valuable. log in screen doesnt bother me as lore spoilers

**Comment by Roxtedy** [comment]
This is a rather ancient mistake that haunts our with TC. Because in AC the translation from TC. Happened because when the TC translation from the 4.3 branch moved to the 3.3.5 branch

**Comment by Defalion** [comment]
> This is a rather ancient mistake that haunts our with TC. Because in AC the translation from TC. Happened because when the TC translation from the 4.3 branch moved to the 3.3.5 branch
> 
> The correct translation of the task only in German and Chinese

is there a fix to this?

**Comment by Maelthyrr** [comment]
I'm sorry but you mentioned using the Single Player Project. Even if they use an AC base, the project gets some custom modifications and we can't offer support for this. We need that you check on a clean AC installation to confirm this issue.

**Comment by Roxtedy** [comment]
> I'm sorry but you mentioned using the Single Player Project. Even if they use an AC base, the project gets some custom modifications and we can't offer support for this. We need that you check on a clean AC installation to confirm this issue.

This is on pure AC, described above by me

**Comment by Roxtedy** [comment]
> > This is a rather ancient mistake that haunts our with TC. Because in AC the translation from TC. Happened because when the TC translation from the 4.3 branch moved to the 3.3.5 branch
> > The correct translation of the task only in German and Chinese
> 
> is there a fix to this?

`UPDATE `quest_template_locale` SET `Details` = 'Надеюсь, ты $gготов:готова; к подвигам, |3-6($c), поскольку для тебя есть дело здесь, в Североземье.$B$BИ я имею в виду не мирные сельские заботы.$B$BСтража Штормграда с трудом поддерживает здесь мир: слишком многие из нас находятся в дальних краях, а тревожных вестей меньше не становится. И потому мы надеемся на помощь всех, кто готов защитить свой дом и своих близких.$B$BЕсли ты $gготов:готова; откликнуться на призыв, то поговори с моим командиром, маршалом Макбрайдом. Он здесь, в аббатстве у меня за спиной.' WHERE `ID` = 783 AND `locale` = 'ruRU';`

**Comment by Roxtedy** [comment]
[Документы маршала Макбрайда](https://www.wowhead.com/wotlk/ru/item=745)

UPDATE `page_text_locale`
SET `Text` = 'РАПОРТ: Кобольды$B$BРядом с Долиной Североземья наблюдается повышенная активность кобольдов. Огромное количество этих существ было замечено к северу от аббатства и на Руднике Горного Эха. Я нашел добровольцев среди паладинов, воинов и всех, кто был готов помочь нам в борьбе с этой напастью.$B$BПока дела идут хорошо. Желающих помочь предостаточно.'
WHERE `ID` = 209 AND `locale` = 'ruRU';

UPDATE `page_text_locale`
SET `Text` = 'ОТЧЕТ: Воры$B$BБолее тревожит то, что не только кобольды, но и банды людей-воров появились в долине. Они носят красные повязки и называют себя братством. Такой уровень организованности может доставить неприятности и здесь, и во всем Элвиннском лесу.$B$BМы идентифицировали одного из их лидеров – это Гаррик Тихокрад – и надеемся вскоре предать его правосудию.'
WHERE `ID` = 210 AND `locale` = 'ruRU';

UPDATE `page_text_locale`
SET `Text` = 'ОТЧЕТ: Волки$B$BАтаки животных на Североземье, о которых я сообщал ранее, увеличились. Эоган считает, что появление волков в долине связано с тем, что их спугнули из Сумеречного леса.$B$BНесмотря на то что волки редко охотятся на людей, растущее их число привело к нескольким стычкам с фермерами. И пропало очень много домашнего скота.'
WHERE `ID` = 211 AND `locale` = 'ruRU';

UPDATE `page_text_locale`
SET `Text` = 'РЕКОМЕНДАЦИЯ:$B$BНоситель данного документа, послуживший Североземью с рвением и отличием, должен быть награжден званием делегата в армии Штормграда. Я уверяю вас, что вы найдете его полезным в Элвиннском лесу.$B$BПодпись:$BМаршал Дуглас Макбрайд,$BАрмия Штормграда, Североземья.'
WHERE `ID` = 212 AND `locale` = 'ruRU';

**Comment by Defalion** [comment]
> fix [Документы маршала Макбрайда](https://www.wowhead.com/wotlk/ru/item=745)
> 
> UPDATE `page_text_locale` SET `Text` = 'РАПОРТ: Кобольды$B$BРядом с Долиной Североземья наблюдается повышенная активность кобольдов. Огромное количество этих существ было замечено к северу от аббатства и на Руднике Горного Эха. Я нашел добровольцев среди паладинов, воинов и всех, кто был готов помочь нам в борьбе с этой напастью.$B$BПока дела идут хорошо. Желающих помочь предостаточно.' WHERE `ID` = 209 AND `locale` = 'ruRU';
> 
> UPDATE `page_text_locale` SET `Text` = 'ОТЧЕТ: Воры$B$BБолее тревожит то, что не только кобольды, но и банды людей-воров появились в долине. Они носят красные повязки и называют себя братством. Такой уровень организованности может доставить неприятности и здесь, и во всем Элвиннском лесу.$B$BМы идентифицировали одного из их лидеров – это Гаррик Тихокрад – и надеемся вскоре предать его правосудию.' WHERE `ID` = 210 AND `locale` = 'ruRU';
> 
> UPDATE `page_text_locale` SET `Text` = 'ОТЧЕТ: Волки$B$BАтаки животных на Североземье, о которых я сообщал ранее, увеличились. Эоган считает, что появление волков в долине связано с тем, что их спугнули из Сумеречного леса.$B$BНесмотря на то что волки редко охотятся на людей, растущее их число привело к нескольким стычкам с фермерами. И пропало очень много домашнего скота.' WHERE `ID` = 211 AND `locale` = 'ruRU';
> 
> UPDATE `page_text_locale` SET `Text` = 'РЕКОМЕНДАЦИЯ:$B$BНоситель данного документа, послуживший Североземью с рвением и отличием, должен быть награжден званием делегата в армии Штормграда. Я уверяю вас, что вы найдете его полезным в Элвиннском лесу.$B$BПодпись:$BМаршал Дуглас Макбрайд,$BАрмия Штормграда, Североземья.' WHERE `ID` = 212 AND `locale` = 'ruRU';

Is this a fix ... or just a confirmation ?

**Comment by Roxtedy** [comment]
> Is this a fix ... or just a confirmation ?

y, its a fix

**Comment by Defalion** [comment]
> > Is this a fix ... or just a confirmation ?
> 
> y, its a fix

sry for newbish questions - how do i implement it? im not a programmer, just recompile?

**Comment by Maelthyrr** [comment]
Non, translations are storage in thé DB. You just need to create a SQL fix. You can find ressources that explain how start here:

- https://www.azerothcore.org/wiki/how-to-create-a-db-pr
- https://www.azerothcore.org/wiki/sql-standards


And here it's an exemple of translation fix:

**Comment by Defalion** [comment]
Hi! It seems that long ago I was the one who opened this issue, but i kind of gave up on the game because of bad translation, i see there were some good work done, so I can fix the game somehow?
